src/cmd_line/subparsers/sort.ts

```diff
@@ -3,15 +3,17 @@ import { Scanner } from '../scanner';
 
 export function parseSortCommandArgs(args: string): node.SortCommand {
   if (!args) {
-    return new node.SortCommand({ reverse: false, ignoreCase: false });
+    return new node.SortCommand({ reverse: false, ignoreCase: false, unique: false });
   }
 
-  let scannedArgs: node.ISortCommandArguments = { reverse: false, ignoreCase: false };
+  let scannedArgs: node.ISortCommandArguments = { reverse: false, ignoreCase: false, unique: false };
   let scanner = new Scanner(args);
   const c = scanner.next();
   scannedArgs.reverse = c === '!';
 
-  scannedArgs.ignoreCase = scanner.nextWord() === 'i';
+  const nextWord = scanner.nextWord();
+  scannedArgs.ignoreCase = nextWord === 'i';
+  scannedArgs.unique = nextWord === 'u';
```

Comment:
Is it possible to do both unique and ignorecase?

Reply:
You're probably right that it's not a common usage. I'll leave it up to you, but at least write a comment mentioning that this case isn't covered. Thanks!